src/js/angular/autocomplete/templates/modal/add-label.html

```diff
@@ -18,12 +19,12 @@ <h3 ng-if="!isNew" class="modal-title">{{'add.label.edit.autocomplete' | transla
 
         <input id="wb-autocomplete-languages" type="text" ng-model="label.languages" class="form-control"
                placeholder="{{'add.label.enter.comma.separated' | translate}}"/>
-    </div>
-    <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" ng-click="cancel()">{{'common.cancel.btn' | translate}}</button>
-        <button id="wb-autocomplete-savegraph-submit" class="btn btn-primary" ng-click="ok()" type="submit">{{'common.ok.btn' | translate}}</button>
-    </div>
-</form>
+    </form>
+</div>
+<div class="modal-footer">
+    <button type="button" class="btn btn-secondary" ng-click="cancel()">{{'common.cancel.btn' | translate}}</button>
+    <button id="wb-autocomplete-savegraph-submit" class="btn btn-primary" ng-click="ok()" type="submit">{{'common.ok.btn' | translate}}</button>
```

Comment:
With these changes, is this the default button now?

Reply:
No.
I fixed by adding `id` attribute to form and `form` attribute to button with value set to form id.
Now when enter is pressed the form is submited